examples/synthesis/gan_evaluation.py

```diff
@@ -39,10 +40,10 @@ def main():
     monai.config.print_config()
     logging.basicConfig(stream=sys.stdout, level=logging.INFO)
     set_determinism(12345)
-    device = torch.device("cuda:0")
+    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
 
     # load generator
-    network_filepath = "./network_final.pth"
+    network_filepath = glob("./model_out/*.pth")[0]
```

Comment:
Why change the model name?

Reply:
the reason here was, when i ran the training file it saved the model in "model_out" folder. There was no './network_final.pth' model file available on running the training script. Can you please check that, if i did that right?

Reply:
Hi @gagandaroach ,

Could you please help double confirm the saved model name in the GAN example? Seems the `training` and `evaluation` programs mismatch?

Thanks.

Reply:
Thanks @arp95 for fixing this. 

For the `evaluation` script I would manually copy the final checkpoint from `trainer` into the base directory as `./network_final.pth`. The proposed changes automatically grab a `.pth` to execute  `evaluation`. 

May I suggest the test ensure `training` went to completion by checking if the final checkpoint is saved. `glob("./model_out/checkpoint_final*.pth")` here in `evaluation`.

Reply:
thanks @gagandaroach Yes will add that test in the script.